### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by Svartstein**
Task list created in an array and displayed by a function that iterates in the list.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Swarzstein , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses the old template to create the readme file, kindly use the [new template](https://github.com/microverseinc/readme-template) to create your readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,58 @@
...
+const getToDoList = () => {
+  if (toDoList === null) {
+    return [];
+  }
+  return toDoList;
+};
 
-  return element;
-}
+// eslint-disable-next-line consistent-return
```
Comment (danmainah) [inline src/index.js]: - [ ] [optional] It's not good practice to disable linter checks, kindly try and fix the error first before disabling it.

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,58 @@
...
+const printList = () => {
+  const list = toDoList;
+  let tasks = '';
+  for (let i = 0; i < list.length; i += 1) {
+    // eslint-disable-next-line no-loop-func
```
Comment (danmainah) [inline src/index.js]: - [ ]  [optional] It's not good practice to disable linter checks, kindly try and fix the error first before disabling it.

File: src/index.js [hunk 3]
```diff
@@ -1,11 +1,58 @@
...
+          </label>
+          <div class="edit-manager flex hidden">
+            <button class="edit-confirm">Confirm</button>
+            <button class="edit-cancel">Cancel</button>
+          </div>
+          <div class="list-editor flex">
+            <button class="edit-task">edit</button>
+            <button class="delete-task">Del</button>
+          </div>
+        </li>
```
Comment (danmainah) [inline src/index.js]: - [ ] @Swarzstein great work on your project however the design doesn't match the recommended one check the comparison below:

Design Given          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Swarzstein , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 3 above)
Comment (danmainah) [reply src/index.js]: - [x] @Swarzstein as previously requested kindly ensure the design matches the design given.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Swarzstein , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (Svartstein) [reply src/index.js]: Literally I have seen other projects and non of them have been rejected because of that.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Swarzstein, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**